.gitignore

```diff
@@ -34,4 +34,8 @@ hs_err_pid*
 out
 target
 *.iml
-.idea
\ No newline at end of file
+.idea
+/gradlew
```

Comment:
These files should be included, please revert. Otherwise, I see 8 changed files instead of two